package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "csde-components-boilerplate",
-  "version": "1.3.0",
+  "version": "1.6.0",
```

Comment:
I'm not sure about this change. I guess boilerplate and validator versions should be the same?